pkg/storage/stores/shipper/downloads/table_manager.go

```diff
@@ -309,6 +317,9 @@ func (tm *TableManager) ensureQueryReadiness(ctx context.Context) error {
 			return err
 		}
 
+		usersToBeQueryReadyLen += len(usersToBeQueryReadyFor)
```

Comment:
I am not sure I see much value with this since the final count would be the sum of all the user indexes from all the tables that are within query readiness. What do you think?

Reply:
For now, yes (so you're right!).

But I was wondering, once we adapt the table manager to only download data from tenants assigned to it, the number of reported users is expected to be different. Don't you think this will be useful to check if the different instances are being assigned a balanced number of tenants? Or do you think there's a better way of doing it?

Reply:
> once we adapt the table manager to only download data from tenants assigned to it, the number of reported users is expected to be different

Is the plan to do this :point_up: soon?